### DISCUSSION TITLE
R Shiny Dashboard

### DISCUSSION THREAD

**Discussion by tleonard6** [neherlab/covid19_scenarios#593] [General]
Hello, this is probably a silly question. I have an R shiny dashboard that I run with IHME and CHIME on the county level for the U.S.. All credit is paid to the original research and references are included. With IHME I'm able to download the data every morning and with CHIME, we just programmed the model into our dashboard. Has anyone discussed adapting the Neher Lab models into R at this point or is there a convenient way to do so? Our current dashboard is below.

https://covid19dash.shinyapps.io/covid19/

Thank you,
Taylor

**Comment by nnoll** [comment]
I'm not sure how easy it is to integrate python into R but the models have been (mostly but not quite fully) implemented in Python within data/scripts/model.py. These are used for our data fitting procedure. Outside of this, src/algorithms/model.ts has most of the javascript modelling code. Hope this helps.
